File: ARDUINO/output_arduino.go
package main

import (
	"time"

	"github.com/argandas/serial"
)

func main() {
	sp := serial.New()
	err := sp.Open("COM3", 9600, time.Second*1)
	if err != nil {
		panic(err)
	}
	defer sp.Close()
	sp.Println("AT")
	sp.WaitForRegexTimeout("OK.*", time.Second*10)
}

File: ARDUINO/output_arduino.py
#This is a simple test code for put the gram in a file
import serial
ser = serial.Serial("COM3",timeout=3)
file = open("output.txt","a") 
while 1:
    donnee=str(ser.readline().decode().strip('\r\n'))
    file.write(donnee) 
    file.write("\n")
    print(donnee)
file.close()

